CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
 # Change log for Microsoft365DSC
 
 # UNRELEASED
-
+* EXODistributionGroup
+  * Initial Release
```

Comment:
Please provide a meaningful description of the change. E.g., "Fixed an error where the name wasn't properly escaped in the Filter.